### PR TITLE
feature(cli) - Support `package` as an alias for the `submit --dry-run` argument in the cfn-cli 

### PR DESCRIPTION
**Pull request by mmaeng**
Issue #286 

Request to create a `package` argument which would be an alias to the `submit --dry-run` argument.  This would be to more accurately describe what is occurring during a `cfn submit --dry-run`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/rpdk/core/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"""This is the same as cfn submit --dryrun, it will create a local package without uploading.
+
+Projects can be created via the 'init' sub command.
```
Comment (kddejong) [inline src/rpdk/core/package.py:3]: ?

File: tests/test_package.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from rpdk.core.project import Project
+
+
+def test_package_command_valid_schema(capsys):
+    mock_project = Mock(spec=Project)
+
+    with patch("rpdk.core.package.Project", autospec=True, return_value=mock_project):
+        main(args_in=["package"])
+
+    mock_project.load.assert_called_once_with()
```
Comment (kddejong) [inline tests/test_package.py:13]: This seems light doesn't it?  Is this actually called twice and should we evaluate the parameters?

Comment (mmaeng) [reply tests/test_package.py]: Added more explicit tests

**Comment by mircealam** [comment]
@mmaeng could you please update this to solve conflicts? thank you.

**Review by mircealam — APPROVED** [review]

**Review by kddejong — APPROVED** [review]

**Pull request merged**